media-libs/libjxl/libjxl-0.8.2.ebuild

```diff
@@ -30,6 +30,10 @@ DEPEND="app-arch/brotli:=[${MULTILIB_USEDEP}]
 	test? ( dev-cpp/gtest )
 "
 RDEPEND="${DEPEND}"
+PATCHES=(
+	"${FILESDIR}/${PN}-0.8.2-backport-pr2596.patch"
```

Comment:
Needs a revbump as the lcms2 fixes are for a real problem exposed by tests, not a test-only issue

Reply:
Done